.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/232181311-8f5146b8-9794-4054-9620-4dd330632e3e.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalize 👍 because some of your commit messages are not capitalized and meaningful 😨 Please consider this issue to make your pull request professional and descriptive ☺️